library/mongo-express

```diff
@@ -1,14 +1,23 @@
 # this file is generated via https://github.com/mongo-express/mongo-express-docker/blob/4b43fe8a1206434cb32a006cd155dd71462f092f/generate-stackbrew-library.sh
 
-Maintainers: Nick Cox <nickcox1008@gmail.com> (@knickers)
+Maintainers: Nick Cox <nickcox1008@gmail.com> (@knickers),
+             John Steel <john@jskw.dev> (@BlackthornYugen)
 GitRepo: https://github.com/mongo-express/mongo-express-docker.git
-
-
-GitCommit: 26e7ea94e4d222de7d52531caee52149ac093c26
-Tags: 1.0.0-alpha.4, 1.0.0-alpha, latest
+GitCommit: a375909f86e1cac3ecc7ce381dc2ef4086f111a9
+Tags: 1.0.0-20-alpine3.18, 1.0-20-alpine3.18, 1-20-alpine3.18
 Architectures: amd64, arm64v8
-
-
-GitCommit: 4b43fe8a1206434cb32a006cd155dd71462f092f
-Tags: 0.54.0, 0.54
+GitCommit: 2a6d82f29978e82ab5685618dc34ba6043568917
+Directory: 1.0/20-alpine3.18
+1.0.0-20 1.0-20 1-20 1.0.0-20-alpine3.17 1.0-20-alpine3.17 1-20-alpine3.17
+Tags: 1.0.0-20, 1.0-20, 1-20, 1.0.0-20-alpine3.17, 1.0-20-alpine3.17, 1-20-alpine3.17
```

Comment:
It looks like there is an extra line above the tags that it is getting hung up on.  Maybe a debug line accidentally left in?

Reply:
Oh right; didn't even notice that before. Let me take it out.

Reply:
Removed in dbc659b46f6d590e76e4c3e4863191a8635a49ce -- I also spaced out the variants so it's easier to read.